backend/danswer/db/chat.py

```diff
@@ -130,6 +131,9 @@ def get_chat_sessions_by_user(
     if deleted is not None:
         stmt = stmt.where(ChatSession.deleted == deleted)
 
+    if limit:
```

Comment:
getting all chat sessions got really slow for someone that had been using for a long time. 

In the future, we need to add pagination on the FE, but this is likely fine for now